gtsfm/scene_optimizer.py

```diff
@@ -225,24 +227,19 @@ def create_computation_graph(
                     matching_regime=matching_regime,
                 )
             )
-            if gt_cameras_graph is not None:
-                metrics_graph_list.append(
-                    dask.delayed(two_view_estimator.aggregate_frontend_metrics)(
-                        report_dict,
-                        self._pose_angular_error_thresh,
-                        metric_group_name="verifier_summary_{}".format(tag),
-                    )
+            metrics_graph_list.append(
```

Comment:
does this work well if gt_cameras == None?

Reply:
Yep. Individual functions are supposed to handle missing GT cameras now.